package.json

```diff
@@ -1,84 +1,54 @@
-    "axios": "^0.19.0",
-    "classnames": "^2.2.5",
-    "d2": "^30.0.4",
-    "eventemitter2": "^5.0.1",
-    "final-form": "^4.18.2",
-    "jszip": "^3.1.5",
-    "material-ui": "^0.20.0",
-    "moment": "^2.22.1",
-    "prop-types": "^15.6.1",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-final-form": "^6.3.3",
-    "react-redux": "^7.0.3",
-    "react-router-dom": "^5.0.0",
-    "react-scripts": "3.0.1",
-    "redux": "^4.0.1",
-    "redux-devtools-extension": "^2.13.8",
-    "redux-thunk": "^2.3.0",
-    "reselect": "^4.0.0",
-    "typeface-roboto": "^0.0.54"
-  },
-  "devDependencies": {
-    "@babel/helper-create-regexp-features-plugin": "^7.7.4",
-    "@dhis2/cli-style": "^3.2.2",
-    "@dhis2/d2-i18n-extract": "^1.0.8",
-    "@dhis2/d2-i18n-generate": "^1.1.1",
-    "concurrently": "^5.0.0",
-    "cypress": "^3.6.1",
-    "cypress-cucumber-preprocessor": "^1.16.0",
-    "enzyme": "^3.9.0",
-    "enzyme-adapter-react-16": "^1.13.2",
-    "enzyme-to-json": "^3.4.0",
-    "husky": "^2.3.0",
-    "query-string": "^6.9.0",
-    "redux-mock-store": "^1.5.3",
-    "wait-on": "^3.3.0"
-  },
-  "husky": {
-    "hooks": {
-      "commit-msg": "d2-style commit check",
-      "pre-commit": "d2-style js apply"
+    "name": "import-export-app",
+    "version": "1.0.0",
+    "description": "",
+    "license": "BSD-3-Clause",
+    "private": true,
+    "scripts": {
+        "build": "d2-app-scripts build",
+        "start": "d2-app-scripts start",
+        "test": "d2-app-scripts test",
+        "prettier-write": "prettier \"./src/{**/*,*}.js\" --write",
+        "lint:js": "d2-style js check",
+        "lint:text": "d2-style text check",
+        "lint:staged": "yarn lint:js --staged && yarn lint:text --staged",
+        "lint": "yarn lint:js && yarn lint:text",
+        "format:js": "d2-style js apply",
+        "format:text": "d2-style text apply",
+        "format:staged": "yarn format:js --staged && yarn format:text --staged",
+        "format": "yarn format:js && yarn format:text",
+        "cypress:open": "yarn d2-utils-cypress open"
+    },
+    "devDependencies": {
+        "@dhis2/cli-app-scripts": "^3.2.5",
+        "@dhis2/cli-style": "^5.1.0",
+        "@dhis2/cli-utils-cypress": "^1.0.2",
+        "@testing-library/jest-dom": "^5.1.0",
+        "@testing-library/react": "^9.4.0",
+        "babel-eslint": "^10.0.3",
+        "babel-preset-env": "^1.7.0",
+        "babel-preset-react-app": "^9.0.2",
+        "babel-preset-stage-0": "^6.24.1",
+        "eslint": "^6.7.2",
+        "eslint-config-prettier": "^6.7.0",
+        "eslint-config-react-app": "^5.0.2",
+        "eslint-plugin-cypress": "^2.7.0",
+        "eslint-plugin-flowtype": "^4.5.2",
+        "eslint-plugin-import": "^2.18.2",
+        "eslint-plugin-jsx-a11y": "^6.2.3",
+        "eslint-plugin-prettier": "^3.1.1",
+        "eslint-plugin-react": "^7.17.0",
+        "eslint-plugin-react-hooks": "^2.3.0",
+        "prettier": "^1.19.1",
```

Comment:
Hey @awgaan - this looks really cool!  I think all these `babel` and `eslint` dependencies should be extraneous now, can we remove lines 28 to 42?

Reply:
Done. e66d3f8